core/services/mongodb-remote/src/main/java/com/mongodb/stitch/core/services/mongodb/remote/sync/internal/NamespaceSynchronizationConfig.java

```diff
@@ -227,6 +236,32 @@ Codec getDocumentCodec() {
     return documentCodec;
   }
 
+  boolean addSynchronizedDocuments(
+      final BsonValue... documentIds
+  ) {
+    Map<BsonValue, CoreDocumentSynchronizationConfig> configs = new HashMap<>();
+    for (final BsonValue documentId : documentIds) {
+      if (getSynchronizedDocument(documentId) == null) {
```

Comment:
[q] Should we add a `getSynchronizedDocuments` to speed this up?

Reply:
I don't think we need to, it's just reading from a hashmap.

Reply:
:+1: